encodings/runend/src/compress.rs

```diff
@@ -101,14 +101,14 @@ fn runend_encode_nullable_primitive<T: NativePType>(
     {
         if e != last {
             ends.push(end);
-            match e {
+            match last {
```

Comment:
this is small but can we rename this `prev`

Reply:
done